ironfish-rust/src/frost_utils/split_secret.rs

```diff
@@ -21,7 +23,7 @@ pub fn split_secret(
     config: &SecretShareConfig,
```

Comment:
can we change this to `pub(crate)` instead of `pub`? No one should call this outside of `ironfish_rust` right?